src/server/index.js

```diff
@@ -231,10 +238,49 @@ nextApp.prepare().then(() => {
       loggedInUsername,
       excludedRepos,
     });
+    const profile = data.profile;
+    const dependenciesFile = await getObjectsMetadata(id); // get the content of dependencies.json on s3
+    const profileOrder = await getProfileOrder(id);
+    // Check if the profile has been saved before
+    if (
+      dependenciesFile &&
+      profileOrder &&
+      profileOrder.triggeredBy !== loggedInUsername
```

Comment:
Where is that coming from?

Reply:
@flickz Why was that resolved?

Reply:
Should be a mistake but `profileOrder.triggeredBy !== loggedInUsername` shouldn't be there again since we're no longer enforcing user to sign in before backing their stack. 

`dependenciesFile && profileOrder ` are checks to confirm the profile has been saved before and user is about to update the profile.